tests/testdata/PMF_EBAS/AT0002R.20171130000000.20180809104500.high_vol_sampler.oc_ec_tc.pm25.3mo.3d.AT05L_hvs_02.AT05L_TU_sunset_001.lev2.nas

```diff
@@ -0,0 +1,141 @@
+109 1001 
```

Comment:
Is it possible to do the tests with a smaller set of testdata / smaller amount of test files?

Reply:
the tests with all files run within 0.244 seconds on my development machine. EBAS is always good for surprises, so I want to be sure to get the this small dataset through testing.